generators/app/prompts.js

```diff
@@ -115,6 +117,26 @@ module.exports = [
     default: false,
     when: props => props.ui !== 'ionic'
   },
+  {
+    type: 'confirm',
+    name: 'i18n',
+    message: 'Do you want language internationalization support?',
+    default: true
+  },
+  {
+    type: 'checkbox',
+    name: 'i18n-langs',
```

Comment:
use `i18nLanguages` for consistency (this way you can also use direct dotted notation when accessing props, ie: `props.i18nLanguages`)